Simplified ChIP-exo assays

Matthew J. Rossi, William K. M. Lai, and B. Franklin Pugh

Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, PA 16802, USA

ChIP-seq and ChIP-exo assays identify where proteins bind along any genome in vivo. Although ChIP-seq is widely adopted in academic research, its inherently high noise has limited its use in biotechnology. In contrast, ChIP-exo has low noise and achieves near-base pair resolution. Consequently, and unlike other genomic assays, ChIP-exo provides structural information on genome-wide binding proteins. Construction of ChIP-exo libraries is technically difficult. Here we describe greatly simplified ChIP-exo methods, each with use-specific advantages. This is achieved through assay optimization and use of Tn5 tagmentation and/or single-stranded DNA ligation. Greater library yields, lower processing time, and lower costs are achieved. In comparing assays, we reveal substantial limitations in other ChIP-based assays.  Importantly, the new ChIP-exo assays allow high-resolution detection of protein-DNA interactions in organs and in as few as 25,000 cells. It is suitable for high-throughput parallelization. The simplicity of ChIP-exo now makes it a highly appropriate substitute for ChIP-seq, and for broader adoption in biotechnology. 
