# Simplified ChIP-exo assays

### Matthew J. Rossi<sup>1</sup>, William K. M. Lai<sup>1</sup>, and B. Franklin Pugh<sup>1</sup>

<sup>1</sup>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, PA 16802, USA

### Correspondence: bfp2@psu.edu

### PMID : [30030442](https://pubmed.ncbi.nlm.nih.gov/30030442/)
### Human & yeast GEO ID : [GSE110681](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE110681)
### Mouse GEO ID : [GSE114606](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114606)

## Abstract
ChIP-seq and ChIP-exo assays identify where proteins bind along any genome in vivo. Although ChIP-seq is widely adopted in academic research, it has inherently high noise. In contrast, ChIP-exo has relatively low noise and achieves near-base pair resolution. Consequently, and unlike other genomic assays, ChIP-exo provides structural information on genome-wide binding proteins. Construction of ChIP-exo libraries is technically difficult. Here we describe greatly simplified ChIP-exo methods, each with use-specific advantages. This is achieved through assay optimization and use of Tn5 tagmentation and/or single-stranded DNA ligation. Greater library yields, lower processing time, and lower costs are achieved. In comparing assays, we reveal substantial limitations in other ChIP-based assays.  Importantly, the new ChIP-exo assays allow high-resolution detection of protein-DNA interactions in organs and in as few as 25,000 cells. It is suitable for high-throughput parallelization. The simplicity of ChIP-exo now makes it a highly appropriate substitute for ChIP-seq, and for broader adoption. 
